src/api/core/warren/backends.py

```diff
@@ -13,3 +13,7 @@
         X_EXPERIENCE_API_VERSION="1.0.3", CONTENT_TYPE="application/json"
     ),
 )
+
+
+class ClientException(Exception):
+    """Raised when the client has a failure."""
```

Comment:
We usually define an `exceptions.py` module (e.g. `warren.exceptions`) and define all exceptions there.

```suggestion
class LRSClientException(Exception):
    """Raised when the LRS client has a failure."""
```